src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class Circle extends Figure {
+    public Circle(String color, double radius) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables and pass them on for better code readability. Also, consider if the `radius` should be generated inside this class or passed from somewhere else according to the task description.